### PR TITLE
Resize default wallpaper to size the same as display for PinePhone and PineTab devices

### PR DESCRIPTION
**Pull request by MaksymPrymierov**
Since PinePhone and PineTab displays resolution is known, resize default wallpaper to size the same as display. This will reduce RAM usage and increase UI performance, since SurfaceFlinger won't scale wallpaper every frame anymore.

GitHub issue: https://github.com/GloDroid/glodroid_manifest/issues/44

### CODE REVIEW

**Comment by rsglobal** [comment]
I've tested your PR and it turns out that pinephone has 512x1024 image by default.
Your overlay does not override it. I suppose overlay folder name may need to be changed.

**Comment by rsglobal** [comment]
Ah, sorry 512x1024 is a source crop
Buffer is:
```
0xf8e4bc63ca10: 4992.00 KiB | 1243 (1248) x 1024 |    1 |        1 | 0xb00 | com.android.systemui.ImageWallpaper#0
```

File: pinephone/device.mk [hunk 1]
```diff
@@ -34,3 +34,6 @@ PRODUCT_PACKAGES += \
 
 PRODUCT_PACKAGES += \
     android.hardware.vibrator@1.0-service.pinephone
+
+DEVICE_PACKAGE_OVERLAYS := \
```
Comment (rsglobal) [inline pinephone/device.mk]: Please use `+=`, with :`=` I've got no navigation icons and lock screen appear.

**Comment by rsglobal** [comment]
Please move PP wallpaper to `drawable-nodpi` folder

File: pinephone/device.mk (hunk 1 above)
Comment (MaksymPrymierov) [reply pinephone/device.mk]: Done.

**Comment by MaksymPrymierov** [comment]
> Please move PP wallpaper to `drawable-nodpi` folder

@rsglobal 

Done.

**Comment by rsglobal** [comment]
@MaksymPrymierov ,
https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/content/res/Configuration.java;l=2332;drc=e3f0cc2cc8954c769df9e9a826c6990921839831

**Comment by rsglobal** [comment]
Please apply wallpaper for all DPI, thus making this change dpi-insensitive.

**Comment by rsglobal** [comment]
@MaksymPrymierov ,

Kindly ping.

**Comment by MaksymPrymierov** [comment]
> @MaksymPrymierov ,
> 
> Kindly ping.

Sorry for the delay, I have updated this.

**Review by rsglobal — APPROVED** [review]

**Pull request merged**